test/verify/check-client

```diff
@@ -37,14 +37,14 @@ class TestClient(testlib.MachineCase):
         self.m_target.execute("hostnamectl set-hostname target")
         # validate on-demand install: this does not work on arch, non-split package
         if self.m_target.image.startswith("debian") or self.m_target.image.startswith("ubuntu"):
-            self.m_target.execute("dpkg --purge cockpit-pcp-dbgsym || true; dpkg --purge cockpit-pcp pcp")
+            self.m_target.execute("dpkg --purge pcp")
         elif self.m_target.image != 'arch':
             # TODO: remove conditional when all images have python3-pcp and a Python PCP bridge
             self.m_target.execute("""
                 if rpm -q python3-pcp; then
-                    rpm --erase --verbose cockpit-pcp pcp python3-pcp
+                    rpm --erase --verbose pcp python3-pcp
```

Comment:
Why is this necessary? AFAICS, the code checks for the .service. I suppose the bridge channel checks for the module, and that somehow interferes?

Or is that because `rpm --erase` would fail with just "pcp" (i.e. you need to explicitly remove the dependency)? in that case I don't understand the "else" branch.